### PR TITLE
Consolidate/Remove non styleguide gray colors

### LINKED ISSUE #4971: Consolidate the Caseflow Grays
Related #4958 
## Our Grays
`#323a45` - $color-gray-dark
`#5b616b` - $color-gray
`#aeb0b5` - $color-gray-light
`#d6d7d9` - $color-gray-lighter
`#e4e2e0` - $color-gray-warm-light

## Convert below to ..
$color-gray-medium `#757575` -> 
$gray-font-color  `#6b7079` ->
### random grays either in commons or in our styles
color: `#5b616b`
color: `#666666`
color: `#6b7079`
color: `#737373`
color: `#757575`
color: `#aeb0b5`
color: `#cccccc` also as `ccc`
color: `#d6d7d9`
color: `#b3b3b3`  
color: `#d9d9d9`
color: `#999999`;

### PR DESCRIPTION
**Pull request by nolawi**
Related #4958 Resolves #4971

### Description
Removes gray colors that are not in the style guide, updates colors to the closest matching. [Including](https://github.com/department-of-veterans-affairs/caseflow/issues/4971#issuecomment-376673518)

### Acceptance Criteria 
- [ ] No visually noticeable changes.  (higher contrast in some of the borders, select dropdown)
screen shot 2018-04-03 at 3 53 26 pm
screen shot 2018-04-03 at 3 45 45 pm

### Testing Plan
1. Test Hearing worksheet and Readers document viewer visually

### CODE REVIEW

File: vendor/assets/stylesheets/_rc_collapse.scss [hunk 1]
```diff
@@ -4,7 +4,7 @@
 * This css has been modified to match the accordion for caseflow.
 */
 
-$text-color: #666;
```
Comment (nolawi) [inline vendor/assets/stylesheets/_rc_collapse.scss]: am not sure where this is used

File: app/assets/stylesheets/_reviewer.scss [hunk 2]
```diff
@@ -7,7 +7,7 @@ $caseflow-background: #f9f9f9;
 $bookmark-blue: #23abf6;
 $bookmark-orange: #f6a623;
 $bookmark-white: #ffffff;
-$bookmark-white-outline: #5b616b;
+$bookmark-white-outline: $color-gray;
```
Comment (mdbenjam) [inline app/assets/stylesheets/_reviewer.scss]: You can actually just delete all the `bookmark-(color_name)` variables. They classes that use them are unused.

File: vendor/assets/stylesheets/_rc_collapse.scss (hunk 1 above)
Comment (mdbenjam) [reply vendor/assets/stylesheets/_rc_collapse.scss]: Doesn't Gina's ticket say to remove #666 and replace it with `$color-gray`? Looks like you've redefined here?

This is used in the accordion component.

File: app/assets/stylesheets/react/_select-dropdown.scss [hunk 3]
```diff
@@ -317,18 +317,18 @@
...
   background-color: #ebf5ff;
   /* Fallback color for IE 8 */
-  background-color: #5B616B;
+  background-color: $color-gray;
   color: white;
 }
 
 .Select-option.is-disabled {
-  color: #cccccc;
+  color: $color-gray;
```
Comment (mdbenjam) [inline app/assets/stylesheets/react/_select-dropdown.scss]: Isn't this supposed to be `color-gray-lighter`

File: app/assets/stylesheets/_main.scss [hunk 4]
```diff
@@ -53,15 +53,15 @@ body {
 
 .cf-logo-name,
 .cf-application-title {
-  color: $color-gray-medium !important;
+  color: $color-gray-dark !important;
```
Comment (mdbenjam) [inline app/assets/stylesheets/_main.scss]: Why are we moving medium to dark?

Comment (nolawi) [reply app/assets/stylesheets/_main.scss]: They are actually very close in color, it needs to be removed if its not on this list #4958

**Pull request merged**